geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/clustering/CentroidManagerGeoWave.java

```diff
@@ -283,12 +291,23 @@ public CentroidManagerGeoWave(
 				CentroidParameters.Centroid.ZOOM_LEVEL,
 				1);
 
-		basicAccumuloOperations = new BasicAccumuloOperations(
-				runTimeProperties.getProperty(GlobalParameters.Global.ZOOKEEKER),
-				runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_INSTANCE),
-				runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_USER),
-				runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_PASSWORD),
-				runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_NAMESPACE));
+		try {
+			basicAccumuloOperations = runTimeProperties.getInstance(
+					CommonParameters.Common.ACCUMULO_CONNECT_FACTORY,
+					this.getClass(),
+					BasicAccumuloOperationsFactory.class,
+					DirectBasicAccumuloOperationsFactory.class).build(
+					runTimeProperties.getProperty(GlobalParameters.Global.ZOOKEEKER),
+					runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_INSTANCE),
+					runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_USER),
+					runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_PASSWORD),
+					runTimeProperties.getProperty(GlobalParameters.Global.ACCUMULO_NAMESPACE));
+		}
+		catch (InstantiationException | IllegalAccessException e) {
```

Comment:
Used in situations such as setup in Mapper/Reducers that only handles IOException.  Rather littering the method signature and pushing the redundant handling else where, I do it here.